tests/test_pyclipper.py

```diff
@@ -392,6 +392,16 @@ def test_paths_scale_from(self):
         assert all(isinstance(k, float) for i in res for j in i for k in j)
 
 
+class TestNonStandardNumbers(TestCase):
+    
+    def test_sympyzero(self):
+        from sympy import Point2D
+        path = [(0,0), (0,1)]
+        path = [Point2D(v) for v in [(0,0), (0,1)]]
```

Comment:
I'd add a check that `0` actually gets converted to `sympy.Zero` so that we can be sure the test tests what we want it to.